helm/harvester/charts/harvester/sandbox/init-harvester

```diff
@@ -1,6 +1,23 @@
-# install CA files
-/opt/harvester/bin/panda_common-install_igtf_ca > /var/log/panda/install_igtf_ca.log 2>&1
+#!/bin/bash
 
-# condor logs
-mkdir -p /var/log/panda/condor_logs
-chmod 777 /var/log/panda/condor_logs
+use_default=1
```

Comment:
Isn't it better to change this to something like

#!/bin/bash
\# install CA files
...
\# condor logs
...

if [[ ! -z "${EXPERIMENT}" ]]; then
    DIR=$(dirname "$0") 
    DIR=$(cd ${DIR} && pwd)    
    init_script=${DIR}/${EXPERIMENT}.init-harvester
    if [ -f ${init_script} ]; then
        echo "Executing ${init_script}"
        bash $init_script
    fi
fi

?

The first part is common for all experiments and it
doesn't have to be specified in each blah.init-harvester.
The same comment to run-harvester-crons.

Reply:
updated.